src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private final Random random = Lottery.RANDOM;
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
why not using that `Random` directly?)

```suggestion
    public Color getRandomColor() {
        int index =  Lottery.RANDOM.nextInt(Color.values().length);
        return Color.values()[index];
```